integration/run/run_test.go

```diff
@@ -1132,7 +1132,7 @@ func TestUsingComputeClasses(t *testing.T) {
 		},
 		{
 			name:              "unsupported-region",
-			testDataDirectory: "./testdata/simple",
+			testDataDirectory: "./testdata/computeclass",
```

Comment:
Oh this looks like the computeclass test you added is missing, Oscar.

Reply:
This test case deploys an application with a single compute class that is not configured to the `local` region. Without this change the app will be deployed without any compute class assigned to it. With this change it explicitly tries to use this compute class by name when deploying to the `local` region, which will cause it to error out since that compute class is not configured for that region